relay-event-normalization/src/replay.rs

```diff
@@ -51,9 +51,9 @@ pub fn validate(replay: &Replay) -> Result<(), ReplayError> {
         .value()
         .ok_or_else(|| ReplayError::InvalidPayload("missing segment_id".to_string()))?;
 
-    if segment_id > u16::MAX as u64 {
+    if segment_id > 720 {
```

Comment:
Is it one hour because every segment is 5 seconds long? Would be nice to make this a documented constant, something like

```rust
/// Max. one hour of five-second segments.
const MAX_SEGMENT_ID: u16 = 60 * 60 / 5;
```

```suggestion
    if segment_id > MAX_SEGMENT_ID {
```